### ISSUE TITLE
Can not install dataset from SeuratData

### ISSUE THREAD

**Issue by lilycanfly** [satijalab/seurat#4103]
Hi,
When I try to installData("bmcite"), it shows the following error:
Using cached data manifest, last updated at 2021-02-18
no access to remote seuratdata repository, unable to install new datasets.

Could you please help me? Thanks.
Lili

![image](https://user-images.githubusercontent.com/32545217/108400786-a08f5a00-71e9-11eb-9804-044e624e0da6.png)

**Comment by mojaveazure** [comment]
Hi Lili,

It looks like you don't have internet access or access to our data repository. Can you run the following in the terminal (not R)?

```bash
ping -c 4 seurat.nygenome.org
```

**Comment by denvercal1234GitHub** [comment]
Hi @mojaveazure, I encountered the same issue. 

I reset the Internet. When I ran ping -c 4 seurat.nygenome.org on the terminal in Mac, it showed:

<img width="456" alt="Screen Shot 2021-02-28 at 6 54 25 PM" src="https://user-images.githubusercontent.com/62179082/109429919-66e2fe00-79f6-11eb-8849-dd512b1bc7a6.png">

The code now no longer has the error (No access...), but it has a long warning as below: 

<img width="822" alt="Screen Shot 2021-02-28 at 6 55 56 PM" src="https://user-images.githubusercontent.com/62179082/109429955-985bc980-79f6-11eb-809e-7fe1e5da8d1d.png">

**When I ran SCTransform():**

```
ifnb.list <- lapply(X = ifnb.list, FUN = SCTransform)
```

**It shows this warning below. Should I be concerned about this "Iteration limit reached" and "Found 62 outliers - those will be ignored in fitting/regularization step"?**

Thank you for your help!

<img width="620" alt="Screen Shot 2021-02-28 at 6 58 49 PM" src="https://user-images.githubusercontent.com/62179082/109430034-099b7c80-79f7-11eb-9523-78369e730be0.png">
